go/dbinfo/dbinfo.go

```diff
@@ -47,14 +45,24 @@ func run(out io.Writer, cfg Config) error {
 	return err
 }
 
+type TableColumn struct {
+	Name       string `json:"name"`
+	Type       string `json:"type"`
+	KeyType    string `json:"keyType,omitempty"`
+	IsNullable bool   `json:"isNullable"`
+	Extra      string `json:"extra,omitempty"`
+}
+
 type TableInfo struct {
-	Name string `json:"name"`
-	Rows int    `json:"rows"`
+	Name    string         `json:"name"`
+	Rows    int            `json:"rows"`
+	Columns []*TableColumn `json:"columns"`
 }
 
 type Info struct {
-	FileType string      `json:"fileType"`
-	Tables   []TableInfo `json:"tables"`
+	FileType        string             `json:"fileType"`
+	Tables          []*TableInfo       `json:"tables"`
+	GlobalVariables *map[string]string `json:"globalVariables"`
```

Comment:
why a reference to a map? maps are already sent around by reference, so I'm confused to why